### PR TITLE
[Job Launcher] Added testnet and mainnet setup

### PR DESCRIPTION
**Pull request by eugenvoronov**
## Description
Added an env variable to choose if the job launcher should use testnets or mainnet.

## Summary of changes
- Added `WEB3_ENV` to `.env`
- Updated web3 service `getSigner` method
- Updated unit tests
- Refactored `createJob` method

## How test the changes
`yarn test`

## Related issues
[Job Launcher - Testnet and mainnet setup#750](https://github.com/humanprotocol/human-protocol/issues/750)

### CODE REVIEW

File: packages/apps/job-launcher/server/src/modules/web3/web3.module.ts [hunk 1]
```diff
@@ -1,10 +1,10 @@
-import { Module } from '@nestjs/common';
+import { Logger, Module } from '@nestjs/common';
 import { Web3Service } from './web3.service';
 import { ConfigModule } from '@nestjs/config';
 
 @Module({
   imports: [ConfigModule],
-  providers: [Web3Service],
+  providers: [Web3Service, Logger],
```
Comment (portuu3) [inline packages/apps/job-launcher/server/src/modules/web3/web3.module.ts:7]: why is logger a provider here?

**Review by portuu3 — CHANGES_REQUESTED** [review]
Add a check of chain id in job service when the jobs are requested

File: packages/apps/job-launcher/server/src/modules/web3/web3.module.ts (hunk 1 above)
Comment (eugenvoronov) [reply packages/apps/job-launcher/server/src/modules/web3/web3.module.ts:7]: Yes, it's redundant

File: packages/apps/job-launcher/server/src/modules/job/job.service.spec.ts
Comment (portuu3) [inline packages/apps/job-launcher/server/src/modules/job/job.service.spec.ts:1]: add a test passing wrong chainId and making sure that the chainId validation is being called

**Review by portuu3 — CHANGES_REQUESTED** [review]

File: packages/apps/job-launcher/server/src/common/enums/web3.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+export enum UserStatus {
```
Comment (portuu3) [inline packages/apps/job-launcher/server/src/common/enums/web3.ts:1]: I think this should not be here

File: packages/apps/job-launcher/server/src/modules/job/job.service.spec.ts
Comment (portuu3) [reply packages/apps/job-launcher/server/src/modules/job/job.service.spec.ts:1]: the same for cvat

**Review by portuu3 — CHANGES_REQUESTED** [review]

File: packages/apps/job-launcher/server/src/modules/job/job.service.spec.ts
Comment (eugenvoronov) [reply packages/apps/job-launcher/server/src/modules/job/job.service.spec.ts:1]: copy paste, will remove

File: packages/apps/job-launcher/server/src/common/enums/web3.ts (hunk 2 above)
Comment (eugenvoronov) [reply packages/apps/job-launcher/server/src/common/enums/web3.ts]: copy paste, will remove

**Review by portuu3 — CHANGES_REQUESTED** [review]

**Review by portuu3 — APPROVED** [review]

**Pull request merged**